config/metac.yaml

```diff
@@ -110,8 +110,11 @@ spec:
         - cstorvolumepolicies.cstor.openebs.io
         - cstorvolumereplicas.cstor.openebs.io
         - cstorbackups.openebs.io
+        - cstorbackups.cstor.openebs.io
         - cstorcompletedbackups.openebs.io
+        - cstorcompletedbackups.cstor.openebs.io
         - cstorrestores.openebs.io
+        - cstorrestores.cstor.openebs.io
```

Comment:
Why the above 3 changes?

Reply:
updated the CRD names in the YAML as per the cstor-operator.yaml.